src/js/pages/jobs/JobsTab.js

```diff
@@ -0,0 +1,65 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {RouteHandler} from 'react-router';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import AlertPanel from '../../components/AlertPanel';
+import DCOSStore from '../../stores/DCOSStore';
+
+class JobsTab extends mixin(StoreMixin) {
+
+  constructor() {
+    super();
```

Comment:
Not sure that we actually need to call `super()`. If it's defensive, let's call it with `...arguments`.